Gem/Settings/ViewModels/ChainListSettingsViewModel.swift

```diff
@@ -9,8 +9,19 @@ struct ChainListSettingsViewModel {
     var title: String {
         return Localized.Settings.Networks.title
     }
-    
+
     var chains: [Chain] {
         return AssetConfiguration.allChains
     }
+
+    func items(for searchText: String) -> [Chain] {
```

Comment:
this needs to be used with import chains filter, separate search view model should work.